test/functional/replace-by-fee.py

```diff
@@ -539,5 +542,31 @@ def test_rpc(self):
         assert_equal(json0["vin"][0]["sequence"], 4294967293)
         assert_equal(json1["vin"][0]["sequence"], 4294967294)
 
+    def test_mempool_replace_arg(self):
+        # ensure that no replacements are allowed with -mempoolreplacement=0
+        self.stop_nodes()
+        self.nodes = self.start_nodes(1, self.options.tmpdir, [["-mempoolreplacement=0"]])
+
+        tx0_outpoint = make_utxo(self.nodes[0], int(1.1*COIN))
+
+        tx1a = CTransaction()
+        tx1a.vin = [CTxIn(tx0_outpoint, nSequence=0)]
+        tx1a.vout = [CTxOut(1*COIN, CScript([b'a']))]
+        tx1a_hex = txToHex(tx1a)
+        tx1a_txid = self.nodes[0].sendrawtransaction(tx1a_hex, True)
+
+        # Extra 0.1 BTC fee double-spend that will fail due to argument being disabled
+        tx1b = CTransaction()
+        tx1b.vin = [CTxIn(tx0_outpoint, nSequence=0)]
+        tx1b.vout = [CTxOut(int(0.9*COIN), CScript([b'b']))]
+        tx1b_hex = txToHex(tx1b)
+        assert_raises_jsonrpc(-26, "txn-mempool-conflict", self.nodes[0].sendrawtransaction, tx1b_hex, True)
+        mempool = self.nodes[0].getrawmempool()
+
+        assert (tx1a_txid in mempool)
```

Comment:
Nit, remove `()`?

Reply:
:+1: assert is a statement, not a function